developers/bindings/index.md

```diff
@@ -692,6 +692,56 @@ It uses the `DiscoveryResultBuilder` to create the discovery result.
+Typically this is a serial number, or an IP or MAC address.
+The representation property is used to auto-ignore discovery results of Things that already exist in the system.
+This can happen, a) if a Thing has been created manually, or b) if the Thing has been discovered separately by two mechanisms e.g. by mDNS, and by NetBios, or UPnP.
+If a new Thing goes online, the auto-ignore service of the inbox checks if the inbox already contains a discovery result of the same type where the existing representation property is identical to the representation property of the newly discovered Thing.
+If this is the case, the Thing in the inbox is automatically ignored.
+The representation property must be declared in the [thing-types.xml](thing-xml.md#representation-property)
+
+When comparing representation properties, the framework checks for matches between the representation property of the newly discovered Thing, and both the `properties` and the `configuration parameters` of existing Things.
+
+If defining a representation property for a bridge, the representation property does not need to be **globally** unique, but only unique within the context of the bridge, so long as the discovery service calls `.withBridge(bridgeUID)` when building the DiscoveryResult. e.g. if bridge A. has child Things with representation properties of 1, 2, and 3, and bridge B. also has child Things with representation properties of 1, 2, and 3, they will not conflict.
+
+```java
+DiscoveryResult result = DiscoveryResultBuilder.create(thingUID)
+  .withProperty("uniqueId", nonUniquePropertyValue)
+  .withBridge(bridgeUID) // bridgeUID plus nonUniquePropertyValue are unique
+  .withRepresentationProperty("uniqueId")
+  .build();
+```
+
+Furthermore, if a Thing has two configuration parameters where each individually is not globally unique, but the combination of the two is unique, one can define an extra property that combines the two:
+
+```java
+String cfgParamValA = "value-of-non-unique-config-param-A";
+String cfgParamValB = "value-of-non-unique-config-param-B";
+String uniquePropVal = String.format("%s-%s", cfgParamValA, cfgParamValB);
+...
+DiscoveryResult hub = DiscoveryResultBuilder.create(thingUID)
+  .withProperty("uniqueId", uniquePropVal)
+  .withRepresentationProperty("uniqueId")
+  .build();
+```
+
+If Things are created manually, the property or configuration parameter that will match the auto discovery representation property must be set.
+In the case that a `property` will be used to match the representation property its value must be set in the Thing handler's `initialize()` method:
+
+```java
+updateProperty("uniqueId", uniquePropVal);
+```
+
+Alternatively in the case that a `configuration parameter` will be used to match the auto discovery representation property it must be declared in `thing-types.xml` and in the Thing handler's `Configuration` class:
```

Comment:
`thing-types.xml` is not specific enough. Config-descriptions can be defined in a separate file. I would say something like:

```suggestion
Alternatively in the case that a `configuration parameter` will be used to match the auto discovery representation property it must be declared in `thing-types.xml` or the config-description and in the Thing handler's `Configuration` class:
```

Reply:
Ok. Done. I also added a link to the `config-description` page.